pkg/plugins/policies/meshaccesslog/api/v1alpha1/validator.go

```diff
@@ -47,7 +48,7 @@ func validateFrom(from []*MeshAccessLog_From) validators.ValidationError {
 
 		defaultField := path.Field("default")
 		if fromItem.GetDefault() == nil {
-			verr.AddViolationAt(defaultField, "must be defined")
+			verr.AddViolationAt(defaultField, validation.MustBeDefined())
```

Comment:
I dried up the messages so they are the same for both policies. Let me know what you think.